geode-redis/src/main/java/org/apache/geode/redis/internal/RedisCommandType.java

```diff
@@ -349,9 +350,17 @@ private RedisCommandType(Executor executor, ParameterRequirements parameterRequi
     this.parameterRequirements = parameterRequirements;
   }
 
-  public void executeCommand(Command command, ExecutionHandlerContext executionHandlerContext) {
+  public RedisResponse executeCommand(Command command,
+      ExecutionHandlerContext executionHandlerContext) {
     parameterRequirements.checkParameters(command, executionHandlerContext);
+
+    if (executor instanceof ResponseExecutor) {
```

Comment:
Could you change "Executor" to have "executeCommandsWithResponse"? For all the old executors we have not converted yet, executeCommandsWithResponse could just call executeCommand and then return null.
That would get rid of this instanceof check.

Reply:
Excellent! Good call.